### ISSUE TITLE
JAWS screen reader does not read state of the dropdown list items [selected /unselected] while traversing list along with state collapsed and expanded.

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#16490]
JAWS screen reader does not read state of the dropdown list items [selected /unselected] while traversing list along with state collapsed and expanded.
Note: JAWS screen reader also does not read associated dropdown label on focus .It reads label after dropdown is expanded post focus click on dropdown.

### Environment Information
Browser: Chrome Version 87.0.4280.88 (Official Build) (64-bit)
Screen Reader: JAWS

### Describe the issue:

JAWS screen reader does not read state of the dropdown list items [selected /unselected] while traversing list along with state collapsed and expanded.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown

![issue](https://user-images.githubusercontent.com/62233982/104699133-47d72800-5738-11eb-9c3b-a5f4546b5896.png)
![dropdown_JAWS_ISSUE](https://user-images.githubusercontent.com/62233982/104888347-9f70d000-5992-11eb-837b-26e542650e4e.PNG)

#### Actual behavior:
JAWS screen reader does not read state of the dropdown list items [selected /unselected] while traversing list along with state collapsed and expanded.

#### Expected behavior:

JAWS screen reader should read state of the dropdown list items [selected /unselected] while traversing list along with state collapsed and expanded.

**Comment by v-ajrai** [comment]
Could you please let  us know when we could get fix for this?

**Comment by gouttierre** [comment]
@v-ajrai - thank you for the feedback - I took a look and had broccoli selected and it cited on 5 of 7 - broccoli - not mentioning the selection of the actual item. To confirm - I will loop in the engineering team.

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by v-ajrai** [comment]
@khmakoto cold you let us know when we could get fix for this

**Comment by v-ajrai** [comment]
any update on this..?

**Comment by v-ajrai** [comment]
Any ETA when we could get fix for this

**Comment by v-ajrai** [comment]
@khmakoto  Could you please let us know when could we get any fix for this

**Comment by smhigley** [comment]
I know this is closed, but if anyone else comes here for info, the selection state announcement is fully controlled by JAWS. There's a bug on Freedom Scientific here, though to set expectations, that behavior is probably unlikely to change soon: https://github.com/FreedomScientific/standards-support/issues/440
